qutip/solver/stochastic.py

```diff
@@ -192,10 +193,30 @@ def wiener_process(self):
         return self._trajectories_attr("wiener_process")
 
     def merge(self, other, p=None):
-        raise NotImplementedError("Merging results of the stochastic solvers "
-                                  "is currently not supported. Please raise "
-                                  "an issue on GitHub if you would like to "
-                                  "see this feature.")
+        if not isinstance(other, StochasticResult):
```

Comment:
It seems that `ssesolve` and `smesolve` results can be merged. I only wanted to double-check that this is intended.

Reply:
No...